.github/workflows/tests.yml

```diff
@@ -31,10 +31,21 @@ jobs:
     - name: Build docs
       run: tox -e docs
 
+  build-snap:
+    name: Build snap
+    runs-on: ubuntu-latest
+    needs: lint-unittests-docs
+    steps:
+    - uses: actions/checkout@v2
+    - uses: snapcore/action-build@v1
+      id: snapcraft
+    - name: Install snap
+      run: sudo snap install --dangerous ${{ steps.snapcraft.outputs.snap }}
+
   func-tests:
     name: Integration tests
     runs-on: self-hosted
-    needs: lint-unittests-docs
+    needs: build-snap
```

Comment:
I think that `tox -e build` could be part of lint-unittests-docs. Functional tests don't use it and it's a little confusing like this.

Reply:
you're right - I thought we were using the built package for func testing, but we're not (`PYTHONPATH={toxinidir}`)